### DISCUSSION TITLE
How to listen/react on disclosure close and open

### DISCUSSION THREAD

**Discussion by pscheit** [tailwindlabs/headlessui#1809] [Help]
Hey there,

are there events fired when the disclosure is opened or close? Really don't need much else than that

```
<Disclosure @onOpen="doSomething">
```

?

**Comment by tordans** [comment]
FYI there is a possible solution at https://github.com/tailwindlabs/headlessui/discussions/1626#discussioncomment-3833754
Did not try it, though.
